train_segmentation_model.py

```diff
@@ -67,6 +67,16 @@ def train(gcp_bucket, config_file, random_module_global_seed, numpy_random_globa
     logs_dir = Path(model_dir, 'logs')
     logs_dir.mkdir(parents=True)
 
+    path_pre_trained_model = None
+    pre_trained_model_config = None
```

Comment:
i think put these two lines inside of the `if` statement so it's more readable.
```
path_pre_trained_model = None
pre_trained_model_config = None
```

Reply:
done